src/common/gpio.rs

```diff
@@ -37,6 +37,8 @@
 
 use crate::{iomuxc, ral};
 
+use cortex_m::interrupt;
```

Comment:
If keeping, prefer the [`critical-section`](https://docs.rs/critical-section/latest/critical_section/) critical section. These drivers can be used on multi-core MCUs, and using `critical-section` better prepares us to support those MCUs.

Reply:
Got it, will do so in the future. Will revert that change for now anyway as I realize it isn't necessary (for now, might become necessary when we implement `eh1-async`)